entropy/unikernel.ml

```diff
@@ -0,0 +1,30 @@
+open Lwt
+
+module Main (C: V1_LWT.CONSOLE) = struct
+
+(*  let sources () =
+    (* some conditional compilation stuff here, Nocrypto_entropy_mirage is only
+       available on Xen and Solo5, but Nocrypto_entropy_unix.sys_rng may be of interest *) 
+    let tostr = function
+      | `Timer -> "timer"
+      | `Rdseed -> "rdseed"
+      | `Rdrand -> "rdrand"
+      | `Xentropyd -> "xentropyd"
+    in
+    match Nocrypto_entropy_mirage.sources () with
+    | Some xs -> C.log_s c ("sources: " ^ String.concat ", " (List.map tostr xs))
+    | None -> C.log_s c "no sources" *)
```

Comment:
Why are these commented out?  We should either include them with comments on why they might not work, or remove them entirely; otherwise the example is confusing.  (I'm confused, for example.)